central/sensor/service/pipeline/nodescansv2/pipeline.go

```diff
@@ -0,0 +1,60 @@
+package nodescansv2
+
+import (
+	"context"
+
+	"github.com/pkg/errors"
+	countMetrics "github.com/stackrox/rox/central/metrics"
+	"github.com/stackrox/rox/central/sensor/service/common"
+	"github.com/stackrox/rox/central/sensor/service/pipeline"
+	"github.com/stackrox/rox/central/sensor/service/pipeline/reconciliation"
+	"github.com/stackrox/rox/generated/internalapi/central"
+	"github.com/stackrox/rox/pkg/logging"
+	"github.com/stackrox/rox/pkg/metrics"
+)
+
+var (
+	log = logging.LoggerForModule()
+)
+
+// Template design pattern. We define control flow here and defer logic to subclasses.
+//////////////////////////////////////////////////////////////////////////////////////
+
```

Comment:
This comment does not make anything clear-er, for me. Unless you see a value in it, I suggest removing it.
```suggestion
```

Reply:
Done